### PR TITLE
Improved DatePickerRange

### PR DESCRIPTION
**Pull request by valentijnnieman**
This improves the DatePickerRange component, with bugfixes for #209 and #152 .

### CODE REVIEW

**Comment by willgdjones** [comment]
Looking forward to this being merged. Thanks for your work @valentijnnieman !

**Comment by willgdjones** [comment]
Any movement on this? Would be great to see this integrated asap. Thanks!

File: dash_core_components/metadata.json [hunk 1]
```diff
@@ -139,73 +139,6 @@
       }
     }
   },
-  "src/components/Confirm.react.js": {
```
Comment (bpostlethwaite) [inline dash_core_components/metadata.json]: Curious why this chunk of code was removed?

Comment (valentijnnieman) [reply dash_core_components/metadata.json]: That is curious... I have no idea why it's there in the first place, as there's no Confirm component or anything. Could it be related to the new Confirmation modal?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.23.1]
+### Fixed
+- Improved DatePickerRange, fixing issues #209 and 152
```
Comment (chriddyp) [inline CHANGELOG.md]: Let's make these links to the issues, i.e.:
> [#209](https://github.com/plotly/dash-core-components/issue/209) and [#152](https://github.com/plotly/dash-core-components/issue/152)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
...
+
+### Added
```
Comment (chriddyp) [inline CHANGELOG.md]: I think we can ⚡️  this one

File: src/components/DatePickerRange.react.js [hunk 4]
```diff
@@ -313,7 +320,17 @@ DatePickerRange.propTypes = {
...
+
+    /**
+     * Determines when the component should update
+     * its value. If `bothdates`, then the DatePicker 
+     * will only trigger its value when the user has
+     * finished picking both dates. If `singledate`, then
+     * the DatePicker will update its value 
+     * as one date is picked.
+     */
+    updatemode: PropTypes.oneOf(['singledate', 'bothdates'])
```
Comment (chriddyp) [inline src/components/DatePickerRange.react.js]: 👍 Nice

**Pull request merged**